.github/actions/split_version/action.yaml

```diff
@@ -0,0 +1,47 @@
+name: 'Split Version'
+description: 'Splits the inputed version into major, minor, and patch components'
+inputs:
+  version: 
+    description: 'Version to split'
+    required: true
+  release-branch:
+    description: 'Release branch'
+    required: true
+  existing-version:
+    description: 'Existing version'
+    required: true
+outputs:
+  minor-version:
```

Comment:
S: Would update this variable to be `major-minor-version`, sort of misleading currently.